### DISCUSSION TITLE
Trying to remove Tails, Knuckles, Amy, and changing Sonic to Jim/J

### DISCUSSION THREAD

**Discussion by Bispait** [Techokami/SonicWorldsNext#105] [General]

**Comment by DimensionWarped** [comment]
Stating your intentions doesn't constitute a discussion. I assume you are wanting to know how to do this, and the process is basically to just reverse the process of adding a new character (look at the code that was used to add Amy and do that opposite of what that was... you should be able to find specific commits for that) -- and then repeat until you get to just Sonic. Then rename Sonic's assets to Jim while keeping in mind all of the other times Sonic is mentioned in the code base. Nothing hard, maybe a little tedious tough.

**Reply by Bispait** [reply]
Would the game crash if I just fully removed all the characters except for sonic?
